src/triples/generation.rs

```diff
@@ -441,6 +445,578 @@ async fn do_generation<C: CSCurve>(
     ))
 }
 
+async fn do_generation_many<C: CSCurve, const N: usize>(
+    ctx: Context<'_>,
+    participants: ParticipantList,
+    me: Participant,
+    threshold: usize,
+) -> Result<TripleGenerationOutputMany<C>, ProtocolError> {
+    assert!(N > 0);
+    
+    let mut rng = OsRng;
+    let mut chan = ctx.shared_channel();
+    let mut transcript = Transcript::new(LABEL);
+
+    // Spec 1.1
+    transcript.message(b"group", C::NAME);
+    transcript.message(b"participants", &encode(&participants));
+    // To allow interop between platforms where usize is different
+    transcript.message(
+        b"threshold",
+        &u64::try_from(threshold).unwrap().to_be_bytes(),
+    );
+    
+    let mut my_commitments = vec![];
+    let mut my_randomizers = vec![];
+    let mut ev = vec![];
```

Comment:
do we wanna use _v here so instead of "ev" it would be "e_v" like in the other places you changed to a vector?  I think we should

Reply:
note: this also applies to the variables below

Reply:
I'll ensure everything is consistent that way then